cypress/integration/app-init.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('App initialization', () => {
+  it('Loads main page', () => {
+    cy.visit('http://localhost:8080/');
+
+    const inputText = 'Movie name';
+
+    cy.get('.search-panel-input')
+      .type(inputText)
+      .should('have.value', inputText);
+
+    cy.get('.search-panel-button')
```

Comment:
This one should be a separate test case. To write `visit` code once please move it to `beforeEach` hook

Reply:
fixed